react/features/base/environment/environment.js

```diff
@@ -81,6 +83,18 @@ export function isSupportedBrowser() {
     return isMobileBrowser() || JitsiMeetJS.isWebRtcSupported();
 }
 
+/**
+ * Returns whether or not the current environment is a supported
+ * browser on a mobile device.
+ *
+ * @returns {boolean}
+ */
+export function isSupportedMobileBrowser() {
+    return (Platform.OS === 'android' && browser.isChrome())
```

Comment:
maybe use isChromiumBased? There are a number of Chromium derivatives already such as Bromite https://www.bromite.org/

Reply:
isChrome check would cover all these chromium based browsers, the only exception being Opera. Is it safe to just enable it for all those browsers w/o testing them ?